src/components/Filters/Filters.types.ts

```diff
@@ -0,0 +1,71 @@
+import PropTypes from 'prop-types';
+
+import { Option, OptionPropType } from './Select/Select.types';
+import { Operators } from './Filters.enums';
+
+type OperatorTypes = typeof Operators[keyof typeof Operators];
+
+interface InputProps {
+  options?: Option[];
+  min?: number;
+  max?: number;
+  length?: number;
+  regexp?: string;
+}
+
+interface InputWithProps {
+  type: React.ReactNode;
+  props: InputProps;
+}
+
+export interface Condition {
+  input: React.ReactNode | InputWithProps;
+  label: string;
+  value: string;
+  isDefault?: boolean;
+}
+
+export interface DataPoint {
+  conditions: Condition[];
+  label: string;
+  value: string;
+}
+
+export interface Filter {
+  operator: OperatorTypes;
+  dataPoint: string;
+  condition: string;
+  input: string;
+}
+
+export interface FiltersProps {
+  dataPoints: DataPoint[];
+  value: Filter[];
```

Comment:
I'm not sure about the propName `value`. One thing is that this is an array, so, the plural is probably more appropriate, like `values`. The other thing is the word itself, maybe `filtersState` or simple `data` would be better

Reply:
I took the name from KendoReact Filter but understand to your concern. I will use `data` as more generic and shorter name